lms/static/styles/_variables.scss

```diff
@@ -1,22 +1,6 @@
-// Colors.
-$white: white;
+$color-text: #202020;
 
-$grey-1: #f2f2f2;
```

Comment:
These values are surfaced as Tailwind utility classes based on the tailwind configuration now.